src/twitter.js

```diff
@@ -91,7 +91,7 @@ class Twitter {
     return json;
   }
 
-  stream(endpoint, parameters) {
+  stream(endpoint, parameters, wait) {
```

Comment:
^ same as the above comment

Reply:
had the options optional instead in he case user does not want to deviate from the defaults